src/utils/efd_eventfd.inc

```diff
@@ -61,18 +69,25 @@ void nn_efd_signal (struct nn_efd *self)
 {
     const uint64_t one = 1;
     ssize_t nbytes;
+    int fd = self->efd;
```

Comment:
Is this for readability, or functionality? Either way, it looks good, but is a slight departure from the rest of the codebase.